### PR TITLE
Add Abacas recipe

### PR DESCRIPTION
**Pull request by svarona**
New recipe create for Abacas program, used for contiguate (align, order, orientate), visualize and design primers to close gaps on shotgun assembled contigs based on a reference sequence.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by svarona** [comment]
@BiocondaBot please add label

File: recipes/abacas/patches/path.patch [hunk 1]
```diff
@@ -0,0 +1,8 @@
+--- abacas.1.3.1.pl	2010-11-03 09:30:45.000000000 -0500
++++ abacas.1.3.1.pl.new	2019-03-27 22:14:40.542475500 -0500
+@@ -1,4 +1,4 @@
+-#!/usr/local/bin/perl
++#!/opt/anaconda1anaconda2anaconda3/bin/perl
```
Comment (bgruening) [inline recipes/abacas/patches/path.patch:5]: You should use `#!/usr/bin/env perl`

File: recipes/abacas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    - perl # [not osx]
+    - mummer
+
+test:
+  commands:
+    - abacas.pl 2>&1 | grep "abacas.pl" > /dev/null
+
+about:
+  home: http://abacas.sourceforge.net/
+  license: GPL-2
```
Comment (bgruening) [inline recipes/abacas/meta.yaml:32]: GPL-2.0-only ?

**Review by bgruening — COMMENTED** [review]
Thanks @svarona 

Thanks review inline. Please do not change .circleci/config.yml

File: recipes/abacas/meta.yaml (hunk 2 above)
Comment (svarona) [reply recipes/abacas/meta.yaml:32]: In the script it says:
`# This program is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License
# as published by the Free Software Foundation; either version 2
# of the License, or (at your option) any later version.`

How do you think it's better to indicate it?

File: recipes/abacas/patches/path.patch (hunk 1 above)
Comment (svarona) [reply recipes/abacas/patches/path.patch]: I think now it's correct, would you mind to check it please?

File: recipes/abacas/meta.yaml [hunk 3]
```diff
@@ -17,8 +17,10 @@ build:
   noarch: generic
 
 requirements:
+  host:
+    - perl # [not osx]
   run:
-    - perl
+    - perl # [not osx]
```
Comment (druvus) [inline recipes/abacas/meta.yaml:23]: I guess it needs perl on both linux and osx?

Comment (svarona) [reply recipes/abacas/meta.yaml]: I've removed the `# [not osx]`, it was a comment dragged form the recipe in another channel.

**Review by druvus — APPROVED** [review]

**Comment by ewels** [comment]
@BiocondaBot please merge

**Pull request merged**